### ISSUE TITLE
Look into including American Samoa, Guam, Northern Mariana Islands, Puerto Rico, US Virigin Islands

### ISSUE THREAD

**Issue by pdil** [pdil/usmap#20]
US territories are frequently excluded from US maps. In the spirit of this package, which is to include often forgotten Alaska & Hawaii, it may be a good idea to allow the option to present the inhabited US territories on the map as well.

These territories all have FIPS codes and official abbreviations so incorporating them into the data set should be seamless.

The difficult part will be moving the coordinates of these places so that they appear on the map in a suitable and aesthetically adequate part of the US map.

**Comment by pdil** [comment]
Example maps for reference since there isn't much precedent in this area:

https://media.gettyimages.com/vectors/map-with-its-territories-vector-id165966331?s=170667a

https://media.gettyimages.com/vectors/map-outline-with-its-territories-vector-id452121583?s=170667a

https://www.rfca.org/portals/0/usa_map_530W.jpg

**Comment by dcaud** [comment]
I'd really appreciate this! I believe [Puerto Rico has > 3,000,000 residents](https://www.nationsonline.org/oneworld/US-states-population.htm), whereas the other territories have less than 200,000. All would be great! But maybe just start with PR and add it inside the Gulf of Mexico?

**Comment by dcaud** [comment]
I've looked at your code a bit and might try to revise it to include PR.

The basic requirements seem to be:

1. Revise [Transform.R](https://github.com/pdil/usmap/blob/27924338c1a7dbd88b26cd85583c3ab15430ee40/R/transform.R) by:

A. duplicated the Hawaii functions with updated coord values for PR.
B. update `# combine all points` to merge in the new PR.

Is that the basics? If so, I might give it a try later today.

**Comment by pdil** [comment]
@dcaud That's correct, I think you got it. Feel free to add a PR and we can work through it to make sure everything is included. Thanks!

**Comment by dcaud** [comment]
I opened a PR and made some changes. I guess we can continue the conversation over there!

**Comment by pdil** [comment]
Yeah the data has since been moved to https://github.com/pdil/usmapdata so the data and scripts will need to be updated there

**Comment by pdil** [comment]
I'm happy to announce that with today's release of `usmapdata 1.0`, Puerto Rico has been added to the data! I will be making the necessary changes to `usmap` and releasing it as version 1.0 as well.
